configs/preselector_config.json

```diff
@@ -1 +1,14 @@
-{}
+{
+	"name": "preselector",
```

Comment:
We may not want to include this file, or the calibration JSON files, or the sensor_definition JSON file.

Reply:
Emptied the preselector config file in [aad728d](https://github.com/NTIA/scos-sensor/pull/218/commits/aad728dd792f2c0ad4755319bffc56a3840cacd2).

Deleted calibration files in [c753810](https://github.com/NTIA/scos-sensor/pull/218/commits/c7538108dc164b5538a345ad7cdd27079b7c4104).

Sensor definition changes are OK.